src/dsp_tools/commands/validate_data/models/validation.py

```diff
@@ -74,7 +75,7 @@ class DetailBaseInfo:
 
 
 @dataclass
-class ValidationResult:
+class ValidationResult(ABC):
```

Comment:
Why is this an ABC? It has no abstract methods that must be overridden in child classes